src/pages/Configuration/Details/index.js

```diff
@@ -115,10 +117,23 @@ export default class ConfigurationDetails extends PureComponent {
         },
         callback: (res) => {
           if (res) {
+            const arr = [];
+            let UpDataList = [];
+            if (info && info.services && info.services.length > 0) {
```

Comment:
What logic?

Reply:
Determines whether component changes take effect

Reply:
Components that should be updated should take a combination of the following two situations:
1. The configuration item has changed, and the effective component should be updated.
2. The component in effect has changed, and the changed component should be updated.

Reply:
It was modified in this way